ansible/roles/test/files/ptftests/lpm.py

```diff
@@ -101,5 +100,7 @@ def ranges(self):
                     interval = self.IpInterval(sorted_boundaries[index], ip_address(u'255.255.255.255'))
                 else:
                     interval = self.IpInterval(sorted_boundaries[index], ip_address(u'ffff:ffff:ffff:ffff:ffff:ffff:ffff:ffff'))
-            ranges.append(interval)
+            # add this to prevent incorrect intervals generated by IpInterval
```

Comment:
could you list an example about the incorrect intervals generated by IpInterval?